MatrixSDK/JSONModels/MXEvent.h

```diff
@@ -160,6 +162,12 @@ typedef enum : NSUInteger
 @property (nonatomic) NSString *sender;
 
 /**
+ Contains the fully-qualified IDs of the users who sent a read receipt for this event.
```

Comment:
Sent read receipts are related to others events than this one. The comment should be updated.
BTW you should add the following note:
"This property is relevant only for events with 'kMXEventTypeStringReceipt' type."